crates/chia-consensus/src/build_compressed_block.rs

```diff
@@ -189,10 +189,7 @@ impl BlockBuilder {
     }
 
     // returns generator, sig, cost
-    pub fn finalize(
-        mut self,
-        constants: &ConsensusConstants,
-    ) -> io::Result<(Vec<u8>, Signature, u64)> {
+    pub fn finalize(mut self, constants: &ConsensusConstants) -> Result<(Vec<u8>, Signature, u64)> {
```

Comment:
I don't see any changes here, except formatting. did `cargo fmt` do this?

Reply:
i think clippy